### PR TITLE
Introducing File property

### PR DESCRIPTION
**Pull request by diego-antonelli**

### CODE REVIEW

File: src/functions.ts [hunk 1]
```diff
@@ -64,6 +64,8 @@ const translateType = (
             return prop.$.type;
         case "widgets":
             return "ReactNode";
+        case "file":
+            return preview && !isChild ? "FileValue" : "DynamicValue<FileValue>";
```
Comment (IIsaku) [inline src/functions.ts]: We are not testing the isChild part anywhere where the outcome would be "DynamicValue<FileValue>"

**Review by IIsaku — CHANGES_REQUESTED** [review]
See my comments. Besides that, I don't see any changes related to the required attribute of file. Is that automatically taken care of? We might want to add a test for that as well.

File: src/functions.ts (hunk 1 above)
Comment (diego-antonelli) [reply src/functions.ts]: Just pushed new tests using a list of files that cover this case

File: tests/outputs/list-files.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    style?: CSSProperties;
+    tabIndex: number;
+}
+
+export interface ActionsType {
+    file: DynamicValue<FileValue>;
+}
+
+export interface ActionsPreviewType {
+    file: FileValue;
```
Comment (IIsaku) [inline tests/outputs/list-files.ts]: This doesn't feel right...

**Review by IIsaku — APPROVED** [review]

**Pull request merged**